### PR TITLE
chore: use base64 url safe encoding for noise

### PR DESCRIPTION
**Pull request by richard-ramos**
Relevant RFC PR:
https://github.com/vacp2p/rfc/pull/575

### CODE REVIEW

**Comment by LNSD** [comment]
We are closing this PR due to the base64 encoding issue resolution: **Payloas must be encoded in regular base64 with padding**.

**Comment by richard-ramos** [comment]
Reopened due to https://github.com/vacp2p/rfc/pull/575#discussion_r1115492012 !

**Comment by LNSD** [comment]
> Reopened due to [vacp2p/rfc#575 (comment)](https://github.com/vacp2p/rfc/pull/575#discussion_r1115492012) !

Sorry for that. In that case, it makes total sense to me ✅

File: waku/v2/protocol/waku_noise/noise_utils.nim [hunk 1]
```diff
@@ -75,11 +75,11 @@ proc digestToSeq*[T](digest: MDigest[T]): seq[byte] =
...
-  qr.add encode(applicationName) & ":"
-  qr.add encode(applicationVersion) & ":"
-  qr.add encode(shardId) & ":"
-  qr.add encode(ephemeralKey) & ":"
-  qr.add encode(committedStaticKey.data)
+  qr.add encode(applicationName, true) & ":"
+  qr.add encode(applicationVersion, true) & ":"
+  qr.add encode(shardId, true) & ":"
+  qr.add encode(ephemeralKey, true) & ":"
+  qr.add encode(committedStaticKey.data, true)
```
Comment (LNSD) [inline waku/v2/protocol/waku_noise/noise_utils.nim:82]:
```suggestion
  qr.add encode(applicationName, safe=true) & ":"
  qr.add encode(applicationVersion, safe=true) & ":"
  qr.add encode(shardId, safe=true) & ":"
  qr.add encode(ephemeralKey, safe=true) & ":"
  qr.add encode(committedStaticKey.data, safe=true)
```

**Review by LNSD — APPROVED** [review]
LGTM ✅

**Pull request merged**